source/installation/test_situations.md

```diff
@@ -1,7 +1,6 @@
-# How to test your changes on "ready to use" situations (for OpenFisca-France)
+# How to test your changes on "ready to use" situations
 
-Often, when making changes to legislation, you need to test it on a situation that works with your country's tax and benefit system.
-Sometimes, these situations can be quite complicated to model (such as roomates).
-Instead of re-writing them everytime, we have pre-packaged a few in a Python Package.
-
-You can find the package along with a usage example in the [Tutorial repository](https://github.com/openfisca/tutorial/tree/master/python/scripts/generate_situation_examples)
+Often, when making changes to legislation it needs testing on a simulation that works with your country's tax and benefit system.
+Sometimes, these simulations can be quite complicated to model.
+Instead of rewriting them everytime, first check your country's repository for `situation_examples` that have already been created. 
```

Comment:
```suggestion
Instead of rewriting them every time, first check your country's repository for `situation_examples` that have already been created. 
```